secureboot_policy.go

```diff
@@ -174,39 +177,39 @@ func (c *winCertificateAuthenticode) wCertificateType() uint16 {
 
 // decodeWinCertificate decodes the WIN_CERTIFICATE implementation from r. Currently supported types are WIN_CERT_TYPE_PKCS_SIGNED_DATA
 // and WIN_CERT_TYPE_EFI_GUID.
-func decodeWinCertificate(r io.Reader) (winCertificate, error) {
+func decodeWinCertificate(r io.Reader) (winCertificate, int, error) {
```

Comment:
this should probably have named return values now, it's not super clear what the int one is, as it is

Reply:
I've named them now - the int is the total number of bytes read